services/common/src/constants/environment.ts

```diff
@@ -14,7 +14,7 @@ export const DEFAULT_ENVIRONMENT = {
   keycloak_url: "https://test.loginproxy.gov.bc.ca/auth",
   flagsmithKey: "4Eu9eEMDmWVEHKDaKoeWY7",
   flagsmithUrl: "https://mds-flags-dev.apps.silver.devops.gov.bc.ca/api/v1/",
-  flagsmithLocal: process.env.FLAGSMITH_ENABLE_LOCAL_EVALUATION || "false",
+  flagsmithLocal: process.env.FLAGSMITH_ENABLE_LOCAL_EVALUATION === "true" || false,
```

Comment:
this was coming in as the string "false" (whether from the env or from it being blank) which == true. Eek.